lib/stormpath-sdk/client.rb

```diff
@@ -20,18 +20,18 @@ class Client
     include Stormpath::Util::Assert
     include Stormpath::Resource::Associations
 
-    attr_reader :data_store, :application
+    attr_reader :data_store, :application, :api_key
```

Comment:
Please make the api_key attribute protected or so, just so we don't expose it to the end user.